airbnb_project/database/airbnb_db_creator.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS listing (
+    airbnb_listing_id VARCHAR(20) PRIMARY KEY,
+    baths INT,
+    beds INT,
+    latitude DECIMAL(8,6), 
+    location VARCHAR(20),
+    longitude DECIMAL(9,6),
+    name VARCHAR(100),
+    person_capacity INT,
+    registration_number VARCHAR(20),
+    room_type VARCHAR(20),
+    title VARCHAR(100)
+);
+
+CREATE TABLE IF NOT EXISTS policy (
+    policy_id SERIAL PRIMARY KEY,
+    policy_name VARCHAR(255),
+    policy_description VARCHAR(255)
+);
+
+CREATE TABLE IF NOT EXISTS listing_policy_result (
+    result_id SERIAL PRIMARY KEY,
+    airbnb_listing_id VARCHAR(50),
+    policy_id INT,
+    policy_result BOOLEAN,
+    result_details VARCHAR(255),
+    result_datetime TIMESTAMP,
+    FOREIGN KEY (airbnb_listing_id) REFERENCES listing(airbnb_listing_id),
+    FOREIGN KEY (policy_id) REFERENCES policy(policy_id)
```

Comment:
I suggest adding `ON DELETE CASCADE` at the end of these two so when a listing or policy is deleted, the result would be deleted as well.